### ISSUE TITLE
#deesil: wrong type of file

### ISSUE THREAD

**Issue by fanyaorong** [EdenBelouadah/class-incremental-learning#5]
there was a question when I run features_extraction.py:
python ./code/features_extraction.py ./config.cf
and my features_extraction.cf is :
________________________________________________________
[features_extraction.py]
num_workers = 6
batch_size = 128
model_load_path = ../model/min_work_deesil.pt
gpu = 1
normalization_dataset_name = cifar100
datasets_mean_std_file_path = ../data/new_data/datasets_mean_std.txt
features_size = 512
used_model_num_classes = 5
images_lists_folder = ../data/cifar-10-batches-py/list_train
destination_dir = ../output/features_train
________________________________________________________
images_lists_folder is a directory as you show in readme but I see that (images_list =cp['images_lists_folder'] ) in this function:
________________________________________________________
train_dataset = ImagesListFileFolder(
        images_list, transforms.Compose([
            transforms.Resize(256),
            transforms.CenterCrop(224),
            transforms.ToTensor(),
            normalize, ]), return_path=True) 
________________________________________________________
images_list have to be  a file directory not a directory.

Did I do something wrong?
thanks a lot if you answer me!

**Comment by EdenBelouadah** [comment]
Hi,
Thank you for pointing out the error. I replaced the 'features_extraction.py' file with the correct one. Please let me know if there is an issue running the code.

**Comment by fanyaorong** [comment]
Thanks you.  
I had done it as you do to it.    :)
